backend/Dockerfile.celery

```diff
@@ -0,0 +1,24 @@
+# Base image
+FROM python:3.9-slim
+
+# Set working directory
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+  unzip \
+  gcc \
+  libffi-dev \
+  musl-dev \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
+# Install Python dependencies
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
I prefer using `uv pip`, it's a way faster package manager for python.
https://github.com/astral-sh/uv

Reply:
I don't know this one, I'll look into it, I already love it because it's written in RUST!